### PR TITLE
Add a way to change the METRICS_WATCH_NAMESPACE which further restric…

### PR DESCRIPTION
**Pull request by jkremser**
note: comma must be escaped when passing the values using `--set` (if it's in the value file, normal unescaped comma works)

```
helm template ./kedify-agent -nkeda \
           --set agent.apiKey=kfy_facefaceface424242d0f7e1963c7d542aee42424242424242424242424242420  \
           --set agent.orgId=42424242-42ab-42ab-42ab-424242424242 \
           --set agent.metricsWatchNamespace="a\,b\,c" | grep -A1 METRICS_WATCH_NAMESPACE
            - name: METRICS_WATCH_NAMESPACE
              value: "a,b,c"
```

### CODE REVIEW

File: kedify-agent/values.yaml [hunk 1]
```diff
@@ -31,6 +31,9 @@ agent:
     pullPolicy: IfNotPresent
   imagePullSecrets: []
 
+  # comma-separated list of namespaces where pod utilization metrics should be collected
+  # or if unset denotes all namespaces, the collecting itself needs to be also enabled by KubernetesMetricsScraping flag
+  metricsWatchNamespace: ""
```
Comment (wozniakjan) [inline kedify-agent/values.yaml:36]: should we consider adding `{{ .Release.Namespace }}` by default?

Comment (jkremser) [reply kedify-agent/values.yaml:36]: yep, that's the plan, basically same logic as in keda upstream helm chart

```
  {{- $namespaces := append (splitList "," .Values.agent.metricsWatchNamespace ) .Release.Namespace -}}
  {{- range $namespaces }}
  ...
```

**Review by zroubalik — APPROVED** [review]
Blind ack

**Comment by jkremser** [comment]
blind merge 🤠

**Pull request merged**